src/js/angular/models/import/checkbox-control-resource-wrapper.js

```diff
@@ -0,0 +1,122 @@
+import {SortingType} from "./sorting-type";
+import {convertToBytes} from "../../utils/size-util";
+
+export class CheckboxControlModel {
```

Comment:
It is a posibility, yes. Although it won't be quite convenient and performant, because the controller works with the array that is returned by the ImportResourceTreeElement#toList method and if you put these functions inside the ImportResourceTreeElement you'd need to call toList internally each time, whereas now it's possible to have a local cache of the `toList` result and just do the filtering, etc